src/DiscordMember.js

```diff
@@ -320,6 +320,8 @@ class DiscordMember {
 
         if (apiUserData.displayName) {
           data.robloxDisplayName = apiUserData.displayName
+        } else {
```

Comment:
displayname still exists and has a value of "" so this block won't trigger.

Reply:
Upon further testing, this is not the case, but I still placed the check in the string formatter instead as that would be better anyways.